src/Hl7.Fhir.Specification.Tests/Snapshot/SnapshotGeneratorTest.cs

```diff
@@ -9786,5 +9786,129 @@ private void logExtensions(string title, IEnumerable<Extension> extensions, int
                     Debug.WriteLine($"{new string(' ', level * 3)}{extension.Url} : {extension.Value}");
             }
         }
+
+        [TestMethod]
+        public async T.Task TestMergingAPreviouslyRemovedElement()
+        {
+            var baseStructureDefinition = new StructureDefinition()
+            {
+                Url = "http://fire.ly/fhir/StructureDefiniton/ObservationBaseLimitedChoiceTypes",
```

Comment:
To prevent large unit test files, it would be better to add these profiles to the folder TestData and use them here.

Reply:
You can use then the already existing `_testResolver` as well.